### PR TITLE
using baskerville

### PR DESCRIPTION
**Pull request by Stephanie-Chou**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
   <title>All Random All Root</title>
   <meta name="description" content="All Random All Root">
   <meta name="author" content="Ryan Haswell">
+  <link rel="stylesheet"
```
Comment (rottencowz) [inline index.html:8]: This gives a 400 error:

400: Missing font family

The requested font families are not available.

Requested: Baskerville (style: normal, weight: 400)

Comment (rottencowz) [reply index.html:8]: https://fonts.google.com/specimen/Libre+Baskerville

and 
`body {
  font-family: 'Libre Baskerville', serif;
 }`

seem to work

**Review by rottencowz — CHANGES_REQUESTED** [review]
Provided link is dead and has no impact on font. Suggest using Libre Baskerville instead.

File: index.html (hunk 1 above)
Comment (Stephanie-Chou) [reply index.html:8]: done

**Review by rottencowz — APPROVED** [review]

**Pull request merged**